mods/modloader.json

```diff
@@ -0,0 +1,322 @@
+{
+    "format": 1,
+    "name": "Modloader",
+    "authors": ["Risugami"],
+    "versions": [
+        {
+            "name": "Modloader",
```

Comment:
`name` should be the versions name (1.2.4) in this case. It should never contain the name of the mod.